RequestAPI/settings-sample.cfg

```diff
@@ -0,0 +1,8 @@
+[default]
+url = https://staging.commcarehq.org/
+password = 
```

Comment:
This is password of what? Could this be made more readable?

Reply:
This Password is used in many of the json files as a value. I'll change it to json_password.